apps/chat/src/components/Chat/ConversationSettingsModels.tsx

```diff
@@ -64,14 +63,29 @@ export const ConversationSettingsModel = ({
     [dispatch, onModelSelect],
   );
 
+  const playbackModelID =
+    conversation.playback?.messagesStack[
+      conversation.playback.activePlaybackIndex
+    ]?.model?.id ?? conversation.model.id;
+
+  const playbackModelName = modelsMap[playbackModelID]?.name || playbackModelID;
+
+  const isPlayback = conversation.playback?.isPlayback;
+
   return (
     <div className="w-full" data-qa="entity-selector">
       <div className="mb-4">{t('Talk to')}</div>
 
       <div className="flex flex-col gap-3" data-qa="recent">
         <div className="grid grid-cols-1 gap-3">
-          {replay.isReplay && (
-            <ReplayAsIsButton replay={replay} conversationId={conversationId} />
+          {conversation.playback?.isPlayback && (
```

Comment:
use `isPlayback` here
```suggestion
          {isPlayback && (
```